### ISSUE TITLE
Creation of Primary Settings for Business.

### ISSUE THREAD

**Issue by amit-k86** [OrderMail/blacklabel#21]
Acceptance Criteria:-
1. User already activated. Creates a New Business.
2. At that time compulsory option of Primary Contact Person & Email Address to be there.
3. Primary Contact Person can be multiple in numbers and can be modified later.

Tasks:-
1. HTML Prototype
2. Review by the Team.
3. Integration with CodeBase.

**Comment by vikasagarwal** [comment]
What I'm doing is:
At the time, a person either creates a business, his/her email address gets added as the primary contact.

**Comment by amit-k86** [comment]
Agreed..!! However Primary Contact & Administrator can be changed afterwards. However Administrators can be multiple in a business but Primary Contact to be Singular..

**Comment by priyankshukla** [comment]
@amit-k86 Are you testing this story? 
If you are, kindly share your observations, so that we will take an appropriate action on this issue.
